sw/device/exts/common/flash_crt.S

```diff
@@ -57,6 +67,7 @@ _start:
   la   a0, _stack_start
   la   a1, _stack_end
   call crt_section_clear
+L_zero_memory_segments_end:
```

Comment:
These labels need to be prefixed with a `.` unless you want them to show up in disassembly etc.

(I need to change the crt assembly to use `.L` prefixes.)

Reply:
Fixed.